modules/terraform-azure-nfs/examples/ex01-nfs_fileshare/variables.tf

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 Cloudera, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+variable "resourcegroup_name" {
+  type = string
+  description = "Resource Group Name"
+}
+
+variable "vnet_name" {
+  type = string
+  description = "Vnet name"
+
+}
+
+
+variable "azure_region" {
+  type = string
+  description = "Region for CDP"
+
+}
+
+variable "env_prefix" {
```

Comment:
Suggest we remove the `env_prefix` input variable for this module - we'll have a specific interface for module and allow the azure-preqs derive the required inputs.

Reply:
removed.